routes/Students.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router = express.Router()
+
+
+const studentInfo = [
+  { id: 1, name: 'Bademus James', age: '23', grade: 'B', contact: 'Adekunle Ajasin' },
+  { id: 2, name: 'Cross Base', age: '25', grade: 'A', contact: '345 fedreckAbel street' },
+  { id: 3, name: 'Zacs Berdex', age: '29', grade: 'D', contact: 'Wanner Bros inert' },
+  { id: 4, name: 'Maset Such', age: '20', grade: 'C', contact: '34, dexter road, close to' }
+];
```

Comment:
Good job so far. This data is only save in memory. To persist data, you should connect to MongoDB Atlas Server online